integreat_cms/cms/templates/pages/page_form.html

```diff
@@ -56,12 +56,12 @@ <h1 class="heading">
         </div>
         <div class="flex flex-wrap flex-col flex-grow pr-2 w-2/3">
             <ul class="flex flex-wrap items-end pl-4">
-                {% for other_language in languages %}
+                {% for other_language in languages|sort_languages:language|slice:":4"%}
                     <li class="mr-1 {% if other_language == language %}z-10{% endif %}" style="margin-bottom: -2px">
                         <div class="bg-white text-blue-500 {% if other_language != language %}hover:bg-blue-500 hover:text-white{% endif %} border-l-2 border-t-2 border-r-2 border-blue-500 font-bold rounded-t-lg">
                             <div class="border-b-2 border-white">
                                 {% if other_language == language %}
-                                    <div class="{{ language.slug }} px-4">
+                                    <div class="{{ language.slug }} pt-2 pb-3 px-4">
```

Comment:
```suggestion
                                    <div class="{{ language.slug }} px-4">
```
We want to keep the language tabs as narrow as they are now, this was explicitly required by the municipality team.